tests/test_models.py

```diff
@@ -4,7 +4,10 @@
 __description__ = "a module that tests models"
 
 
+from unittest.mock import patch, mock_open
+
 from hop import models
+from hop import publish
```

Comment:
Looks like this isn't used based on `make lint`.

Reply:
Thanks, I had missed that. Removed this in the latest commit.